### PR TITLE
Get user key in service node startup

### PR DESCRIPTION
**Pull request by maksimfedin**
user key is now being generated with service node startup in the PROFILE_PATH path.

### CODE REVIEW

File: service-node/src/main/scala/org/enterprisedlt/fabric/service/node/ServiceNode.scala [hunk 1]
```diff
@@ -35,6 +36,7 @@ object ServiceNode extends App {
     // Org variables
     private val OrgName = Option(Environment.get("ORG")).getOrElse(throw new Exception("Mandatory environment variable missing ORG!"))
     private val Domain = Option(Environment.get("DOMAIN")).getOrElse(throw new Exception("Mandatory environment variable missing DOMAIN!"))
+    private val AdminPassword = Option(Environment.get("ADMIN_PWD")).getOrElse(throw new Exception("Mandatory environment variable missing ADMIN_PWD!"))
```
Comment (apolubelov) [inline service-node/src/main/scala/org/enterprisedlt/fabric/service/node/ServiceNode.scala]: this must be optional

File: service-node/src/main/scala/org/enterprisedlt/fabric/service/node/ServiceNode.scala [hunk 2]
```diff
@@ -57,7 +59,7 @@ object ServiceNode extends App {
         DockerSocket,
         LogFileSize,
         LogMaxFiles)
-    private val cryptoManager = new FileBasedCryptoManager(organizationConfig, "/opt/profile/crypto")
+    private val cryptoManager = new FileBasedCryptoManager(organizationConfig, "/opt/profile", AdminPassword)
```
Comment (apolubelov) [inline service-node/src/main/scala/org/enterprisedlt/fabric/service/node/ServiceNode.scala]: any reason to pass profile directory here? pls revert

File: service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FileBasedCryptoManager.scala [hunk 3]
```diff
@@ -24,44 +24,50 @@ import org.slf4j.LoggerFactory
   */
 class FileBasedCryptoManager(
     organizationConfig: OrganizationConfig,
-    rootDir: String
+    rootDir: String,
+    adminPassword: String
 ) extends CryptoManager {
     private val logger = LoggerFactory.getLogger(this.getClass)
     private val orgFullName = s"${organizationConfig.name}.${organizationConfig.domain}"
+    private val cryptoDir = s"$rootDir/crypto"
```
Comment (apolubelov) [inline service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FileBasedCryptoManager.scala]: you can just rename rootDir -> cryptoDir

File: service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FileBasedCryptoManager.scala [hunk 4]
```diff
@@ -24,44 +24,50 @@ import org.slf4j.LoggerFactory
...
-            loadCertAndKey(s"$rootDir/ca/ca"),
-            loadCertAndKey(s"$rootDir/tlsca/tlsca"),
-            loadCertAndKey(s"$rootDir/users/admin/admin")
+            loadCertAndKey(s"$cryptoDir/ca/ca"),
+            loadCertAndKey(s"$cryptoDir/tlsca/tlsca"),
+            loadCertAndKey(s"$cryptoDir/users/admin/admin")
         )
     }
 
+    logger.info(s"Obtaining user key for admin ...")
```
Comment (apolubelov) [inline service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FileBasedCryptoManager.scala]: Pls adjust comment to something more appropriate

File: service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FileBasedCryptoManager.scala [hunk 5]
```diff
@@ -24,44 +24,50 @@ import org.slf4j.LoggerFactory
...
+    val key: KeyStore = getFabricUserKeyStore("admin", adminPassword)
+    key.store(new FileOutputStream(s"$rootDir/admin-${organizationConfig.name}.p12"), adminPassword.toCharArray)
```
Comment (apolubelov) [inline service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FileBasedCryptoManager.scala]: this has to be optional and performed only if admin password set

File: bin/fabric-service-start.sh [hunk 6]
```diff
@@ -20,6 +20,7 @@ serviceID=`docker run -d \
  -e "CERTIFICATION_DURATION=${CERTIFICATION_DURATION}" \
  -e "SERVICE_BIND_PORT=${SERVICE_BIND_PORT}" \
  -e "SERVICE_EXTERNAL_ADDRESS=${SERVICE_EXTERNAL_ADDRESS}" \
+ -e "ADMIN_PWD=${ADMIN_PWD}" \
```
Comment (apolubelov) [inline bin/fabric-service-start.sh]: lets use PASSWORD to not confuse with anything else

File: fabric-service-start-dev.sh [hunk 7]
```diff
@@ -24,6 +24,7 @@ serviceID=`docker run -d \
  -e "SERVICE_BIND_PORT=${SERVICE_BIND_PORT}" \
  -e "SERVICE_EXTERNAL_ADDRESS=${SERVICE_EXTERNAL_ADDRESS}" \
  -e "DOCKER_SOCKET=unix:///host/var/run/docker.sock" \
+ -e "ADMIN_PWD=${ADMIN_PWD}" \
```
Comment (apolubelov) [inline fabric-service-start-dev.sh]: same as above

**Review by apolubelov — CHANGES_REQUESTED** [review]

File: fabric-service-start-dev.sh (hunk 7 above)
Comment (maksimfedin) [reply fabric-service-start-dev.sh]: renamed.

File: bin/fabric-service-start.sh (hunk 6 above)
Comment (maksimfedin) [reply bin/fabric-service-start.sh]: renamed.

File: service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FileBasedCryptoManager.scala (hunk 4 above)
Comment (maksimfedin) [reply service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FileBasedCryptoManager.scala]: adjusted.

**Review by apolubelov — APPROVED** [review]

**Pull request merged**